src/docs/history.adoc

```diff
@@ -2,7 +2,11 @@
 
 Please read release notes at https://github.com/grails/grails-spring-security-oauth2/releases for checking release information in the future.
 
-* Version 3.0.0
```

Comment:
@puneetbehl 🤔 Version 3.0.0 hasn't been released yet, has it?

Reply:
Yes, it isn't.